code/datums/uplink/devices and tools.dm

```diff
@@ -125,7 +125,7 @@
 
 /datum/uplink_item/item/tools/syndieborg
 	name = "Syndicate Cyborg Teleporter"
-	item_cost = 35
+	item_cost = 30
```

Comment:
I don't know how I feel about allowing traitors to have a syndieborg. They're really fucking powerful.